components/frontend/src/source/SourceEntities.test.js

```diff
@@ -134,6 +138,15 @@ it('sorts the entities by integer', async () => {
     assertOrder(["B", "A", "C"])
 })
 
+it('sorts the entities by integer percentage', async () => {
```

Comment:
Perhaps use the numbers "1", "42" and "9", to make sure it doesn't secretly sort alnum